src/app/georeferencer/qgsgeorefmainwindow.cpp

```diff
@@ -1069,6 +1076,26 @@ QLabel *QgsGeoreferencerMainWindow::createBaseLabelStatus()
 
 void QgsGeoreferencerMainWindow::createStatusBar()
 {
+  // add a widget to show/set current rotation
+  mRotationLabel = createBaseLabelStatus();
+  mRotationLabel->setObjectName( QStringLiteral( "mRotationLabel" ) );
+  mRotationLabel->setText( tr( "Rotation" ) );
+  mRotationLabel->setToolTip( tr( "Current clockwise map rotation in degrees" ) );
```

Comment:
```suggestion
  mRotationLabel->setToolTip( tr( "Map rotation in degrees clockwise" ) );
```

I think "current" doesn't add a lot and having "map rotation" as first words helps to understand quickly

Reply:
I kept the tooltip the same as in the main canvas' rotation widget for translation economy. If I indeed change it, the word 'display' could be added so it is clear that the rotation has no effect on georeferencing!